src/lib/MSPVTX/freqTable.h

```diff
@@ -0,0 +1,65 @@
+#include "common.h"
+#include "targets.h"
+
+#define BAND_NAME_LENGTH                8
+#define IS_FACTORY_BAND                 1
+#define CHANNEL_COUNT                   8
+#define FREQ_TABLE_SIZE 48
+
+#define RACE_MODE                       2
+#define RACE_MODE_POWER                 14 // dBm
+#define NUM_POWER_LEVELS                5
+#define POWER_LEVEL_LABEL_LENGTH        3
+
+const uint8_t channelFreqLabel[FREQ_TABLE_SIZE] = {
+    'B', 'A', 'N', 'D', '_', 'A', ' ', ' ', // A
+    'B', 'A', 'N', 'D', '_', 'B', ' ', ' ', // B
+    'B', 'A', 'N', 'D', '_', 'E', ' ', ' ', // E
+    'F', 'A', 'T', 'S', 'H', 'A', 'R', 'K', // F
+    'R', 'A', 'C', 'E', ' ', ' ', ' ', ' ', // R
+    'R', 'A', 'C', 'E', '_', 'L', 'O', 'W', // L
+};
+
+const uint8_t bandLetter[6] = {'A', 'B', 'E', 'F', 'R', 'L'};
+
+uint16_t channelFreqTable[FREQ_TABLE_SIZE] = {
+    5865, 5845, 5825, 5805, 5785, 5765, 5745, 5725, // A
+    5733, 5752, 5771, 5790, 5809, 5828, 5847, 5866, // B
+    5705, 5685, 5665, 5645, 5885, 5905, 5925, 5945, // E
+    5740, 5760, 5780, 5800, 5820, 5840, 5860, 5880, // F
+    5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917, // R
+    5333, 5373, 5413, 5453, 5493, 5533, 5573, 5613  // L
```

Comment:
We might need to check the lowrace frequencies are correct.  I miss read this issue as race band and not lowrace :| https://github.com/OpenVTx/OpenVTx/issues/62 and there appears to be some differences.  Im yet to find the ground truth and what freqs to use.

Reply:
These are the values from Steadyview-X manual, they are also the same on the rapidFIRE
5362, 5399, 5436, 5473, 5510, 5547, 5584, 5621

Reply:
I can change it no probs together with the refactor, just let me know which values to keep 😏

Reply:
Righto, lets go with the number PK posted from SZ and RF.

Reply:
done, refactored freq table to common place and fixed the lowband frequencies